default/stringtables/en.txt

```diff
@@ -9553,22 +9553,22 @@ BLD_TERRAFORM
 Terraforming
 
 BLD_TERRAFORM_DESC
-Terraforms a world one step closer to the environmental preference of the inhabiting species.
+Terraforms a world one step closer to the [[encyclopedia ENVIRONMENT_TITLE]]al preference of the inhabiting species.
 
 BLD_TERRAFORM_REVERT
 Terraforming Reversion
 
 BLD_TERRAFORM_REVERT_DESC
-Terraforms a world one step back towards its original type.
+Terraforms a world one step back towards its original type of [[encyclopedia ENVIRONMENT_TITLE]].
 
 BLD_REMOTE_TERRAFORM
 Remote Terraforming
 
 BLD_REMOTE_TERRAFORM_DESC
-Terraforms a world one step closer to the environmental preference of the inhabiting species. This process can be enacted at an [[encyclopedia OUTPOSTS_TITLE]].
+Terraforms a world one step closer to the [[encyclopedia ENVIRONMENT_TITLE]]al preference of the inhabiting species. This process can be enacted at an [[encyclopedia OUTPOSTS_TITLE]].
```

Comment:
Changes done (no -al)- on line 5291 and 9556 ( [[ENVIRONMENT_TITLE]] instead of [[ENVIRONMENT_TITLE]]al )

I tried for the first time to update the PR with Git (to not add an additional and useless commit), it worked, but it didn't mark your comment as based on an outdated diff.
Maybe, I did something wrong with the push to my remote branch repo. Well, I'll try differently next time.

What I did:

- I modified en.txt in my local branch "en-txt_cleaning"
- git checkout en-txt_cleaning
- git add default/stringtables/en.txt
- git commit --amend --no-edit
- git push -f origin en-txt_cleaning

What's wrong? push -f?

Thanks.